pkg/piecestore/rpc/client/main.go

```diff
@@ -0,0 +1,183 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
```

Comment:
What's the purpose of this program? If it's an example code, it should go under `/examples`. If it target's end users or is necessary for operating the Storj network, it should go under `/cmd`.

Reply:
@aleitner